### ISSUE TITLE
Polca: paired-end reads are mapped as single-end

### ISSUE THREAD

**Issue by ArielSSchwartz** [alekseyzimin/masurca#189]
When using `polca.sh` the resulting bam file includes reads mapped as single-end only.
Is that done intentionally?

**Comment by alekseyzimin** [comment]
Yes, it is done intentionally because we found no significant benefit to using pairing in mapping
